tdrs-frontend/src/components/PrivateRoute/PrivateRoute.js

```diff
@@ -33,12 +40,22 @@ function PrivateRoute({ children, title }) {
     }
   }, [authenticated, authLoading, dispatch, navigate])
 
-  return authenticated ? (
-    <PrivateTemplate title={title}>
-      {children}
-      <IdleTimer />
-    </PrivateTemplate>
-  ) : null
+  if (authenticated) {
+    return (
+      <PermissionGuard
+        requiresApproval={requiresApproval}
+        requiredPermissions={requiredPermissions}
+        notAllowedComponent={<Navigate to="/home" />}
+      >
+        <PrivateTemplate title={title}>
+          {children}
+          <IdleTimer />
+        </PrivateTemplate>
+      </PermissionGuard>
+    )
+  }
+
+  return null
```

Comment:
Unclear on what this is? I'm not JS-native but this seems odd.

Reply:
react components have to return a JSX element or `null`. returning `undefined` (or not returning anything) throws an error. returning `null` just renders nothing, which is what we want in this case.

this entire change just changes from a ternary operator to a straight `if` statement. I was going to add a nested `if`, hence breaking it up, but opted for a component instead and just didn't change it back to a ternary. open to it, but personally i find a regular `if` to be cleaner for this case.